src/services/prebid.js

```diff
@@ -18,10 +18,12 @@ export function queuePrebidCommand(fn) {
  * @param {function} cb - An optional callback function that should fire whenever the bidding has concluded.
  **/
 export function fetchPrebidBids(ad, code, timeout, info, prerender, cb = null) {
+  pbjs.addAdUnits(info);
   pbjs.requestBids({
     timeout,
     adUnitCodes: [code],
-    bidsBackHandler: () => {
+    bidsBackHandler: (result) => {
+      console.log('Bid Back Handler', result);
```

Comment:
this was added for debugging purposes to make sure header bids are being returned to the client properly, I can wrap this in some kind of debug flag if necessary but I found it so helpful that I think it might be worthwhile leaving it here